### PR TITLE
RequestChanges

### PR DESCRIPTION
**Pull request by Anthony-ferrari**
merged all PR changes. changed my routes to remove user id when possible and used auth middleware for protected routes. made minor changes to navbar and also animal card selection -- id to id.id

### CODE REVIEW

File: client/src/Components/forms/UserProfileUpdateForm.js [hunk 1]
```diff
@@ -18,15 +18,15 @@ const UserProfileUpdateForm = (props) => {
                     passwordConfirm: response.data[0].password
```
Comment (jung8) [inline client/src/Components/forms/UserProfileUpdateForm.js:18]: I think we will have the same issue as the others that the password will return a hash.

Comment (Anthony-ferrari) [reply client/src/Components/forms/UserProfileUpdateForm.js]: fixed this in the last commit

**Review by jung8 — APPROVED** [review]
Thanks for the code clean-up and refactor! 👍

File: client/src/Components/forms/UserProfileUpdateForm.js (hunk 1 above)
Comment (jung8) [reply client/src/Components/forms/UserProfileUpdateForm.js]: 👍

**Pull request merged**